connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -531,10 +531,7 @@ class DataFrameReader private[sql] (sparkSession: SparkSession) extends Logging
    */
   @scala.annotation.varargs
   def textFile(paths: String*): Dataset[String] = {
-    // scalastyle:off throwerror
-    // TODO: this method can be supported and should be included in the client API.
-    throw new NotImplementedError()
-    // scalastyle:on throwerror
```

Comment:
should we add an assert on schema to match with the regular Spark path?

Reply:
What does that mean?

Reply:
Meaning that adding:

```diff
    def textFile(paths: String*): Dataset[String] = {
    + assertNoSpecifiedSchema("textFile")
      text(paths : _*).select("value").as[String](sparkSession.implicits.newStringEncoder)
    }
```

check.

Reply:
Done